walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,134 @@
+            }
+            
+            if(event_array[0].getWhen().end() > event_array[1].getWhen().end()){
+                ending_time = event_array[0].getWhen().end();
+            }
+            else{
+                ending_time = event_array[1].getWhen().end();
+            }
+
+            listtransfer.add(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, beginning_time, false));
+            listtransfer.add(TimeRange.fromStartEnd(ending_time, TimeRange.END_OF_DAY, true));
+        }
+        else if(event_array[0].getWhen().overlaps(event_array[1].getWhen()) == false){
+            int beginning_time= event_array[0].getWhen().start();
+            int begend_time = event_array[0].getWhen().end();
+            int endbeg_time = event_array[1].getWhen().start();
+            int ending_time = event_array[1].getWhen().end();
+
+            listtransfer.add(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, beginning_time, false));
+            listtransfer.add(TimeRange.fromStartEnd(begend_time, endbeg_time, false));
+            listtransfer.add(TimeRange.fromStartEnd(ending_time, TimeRange.END_OF_DAY, true));
+        }
+        return_value = listtransfer;
+    }
+
+
+    else if(events.size()>= 2 && request.getAttendees().size() <= 1){
+        if(event_array[0].getWhen().overlaps(event_array[1].getWhen()) == true){
+            int beginning_time = event_array[0].getWhen().start();
+            int ending_time = event_array[1].getWhen().end();
+            int wholeday_counter =0;
+
+            for(int i = 0; i < events.size(); i++){
+                if(event_array[i].getWhen().duration() ==  TimeRange.WHOLE_DAY.duration()){
+                    wholeday_counter++;
+                }
+            }
+            if(wholeday_counter == events.size()){
+                listtransfer.clear();
+            }
+            else{
+                listtransfer.add(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, beginning_time, false));
+                listtransfer.add(TimeRange.fromStartEnd(ending_time, TimeRange.END_OF_DAY, true));
+            }
+            
+        }
+        else if(event_array[0].getWhen().overlaps(event_array[1].getWhen()) == false){
+            int beginning_time= event_array[0].getWhen().start();
+            int begend_time = event_array[0].getWhen().end();
+            int endbeg_time = event_array[1].getWhen().start();
+            int ending_time = event_array[1].getWhen().end();
+
+            long freeSpace = ((event_array[1].getWhen().start()) - (event_array[0].getWhen().end()));
+
+            if (request.getDuration() > freeSpace){
+                listtransfer.clear();
+            }
+            else{
+                listtransfer.add(TimeRange.fromStartDuration(begend_time, request_duration));
+            }
+        }
+        return_value = listtransfer;
+    }
+    
+
+    else{
+        throw new UnsupportedOperationException("TODO: Implement this method");
+    }
```

Comment:
If I understand the problem correctly, this should never be called. So this should be unnecessary.